metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran.py

```diff
@@ -96,8 +100,10 @@ def _extend_lineage(self, connector: Connector, datajob: DataJob) -> Dict[str, s
                 self.report.info(
                     title="Guessing source platform for lineage",
                     message="We encountered a connector type that we don't fully support yet. "
-                    "We will attempt to guess the platform based on the connector type.",
-                    context=f"{connector.connector_name} (connector_id: {connector.connector_id}, connector_type: {connector.connector_type})",
+                    "We will attempt to guess the platform based on the connector type. "
+                    "Note that we use connector_id as the key not connector_name which you may see in the UI of Fivetran. ",
+                    context=f"connector_name: {connector.connector_name} (connector_id: {connector.connector_id}, connector_type: {connector.connector_type})",
+                    log_type=StructuredLogType.LINEAGE,
```

Comment:
I’m not convinced that introducing this enum is making the log entries more self-serve. While there’s definitely room to improve our log messages so they’re more informative and actionable, this kind of categorization doesn’t solves that.

The structured report feature was meant to highlight certain log traces in the UI. Creating categories would make sense if we were planning further UI/UX enhancements: like using different colors, icons, etc. Unless there are concrete plans, or signals from UX experts indicating that this is needed, I’d suggest holding off on adding this for now.